### DISCUSSION TITLE
i18n routing - add an option to disable redirect from root to `defaultLocale` when `prefixDefaultLocale` is set to `true`

### DISCUSSION THREAD

**Discussion by dallyh** [withastro/roadmap#789] [Proposal]
Hello,

as discussed here: https://github.com/withastro/astro/issues/9300#issue-2025501902 it would be great to have the option to opt-out of the redirect that happens from root ("/") to `defaultLocale` (for example "/cs/") when `prefixDefaultLocale` is set to `true`. Currently with the new i18n routing there is no option to do so, which blunders the ability to customize the root page. I think that the issue explains it in details, if some more information is required then I'll be happy to provide it.

Thanks

**Comment by millette** [comment]
[I'd like that too](https://github.com/withastro/astro/issues/9395).

**Comment by millette** [comment]
Yay! <https://astro.build/blog/astro-420/#new-redirecttodefaultlocale-config-option>
